src/js/components/Grommet/doc.js

```diff
@@ -37,9 +37,14 @@ export const doc = Grommet => {
       theme.`,
     ),
     userAgent: PropTypes.string.description(
-      `User agent used to detect the device width for setting the initial 
+      `User agent used to detect the device width for setting the initial
       breakpoint.`,
     ),
+    containerRoot: PropTypes.object.description(
```

Comment:
Thoughts about renaming the prop to `containerTarget`? It will be more consistent with the overall grommet naming.

Reply:
That works for me, I can get it change in all of the places.